Automattic/wp-calypso
# Design Picker Cleanup: delete unused design-setup-site step from signup framework

Related to:

- https://github.com/Automattic/wp-calypso/issues/65847

## Proposed Changes

The `design-setup-site` step is no longer used as we have moved all the logic to the `designSetup` step in the Stepper Framework.

## Testing Instructions

Do smoke test on various onboarding flows (e.g. `/setup/site-setup` and `/start/do-it-for-me`) and verify they still work.

## Pre-merge Checklist
